### ISSUE TITLE
Canadian Highlander

### ISSUE THREAD

**Issue by fenhl** [taw/magic-search-engine#71]
Addition of [Canadian Highlander](https://canadianhighlander.wordpress.com/) as a format.

The biggest problem here is the unique [points list](https://canadianhighlander.wordpress.com/rules-the-points-list-and-deck-construction/) system for restricted cards. Format information has to be parsed from the website. Ideally, the legality information logic would also be updated to show the number of points for each restricted card.

I will likely try to implement this once I have more time, so feel free to assign to me.

**Comment by taw** [comment]
I'm reluctant to implement unofficial formats (unless they're popular). Every such format means that someone needs to update its banlist for the rest of forever, and right now [every card displays list of formats it's legal in](http://mtg.wtf/card/m11/149/Lightning-Bolt).

Especially if format needs a point system, or otherwise needs special support.

There are already a few formats of questionable usefulness in.

**Comment by taw** [comment]
Are you still interested in it?

Actually maybe we should cleanup the formats we support.

Official formats are all in one place, Commander we definitely need, and Penny Dreadful is basically automated.

I'm not sure if I want to keep "Duel Commander" and "MTGO Commander".

**Comment by fenhl** [comment]
Yes, I'm still interested, there's just more important features I'm focusing on.

If you were to drop any formats, I'd continue to maintain them on my fork.

**Comment by taw** [comment]
That reminds me, is MTGO 1v1 Commander still a thing? It's not included in normal banlist announcements. Also Historic is sort of a format now.

Canadian Highlander would require too many changes to happen fairly obscure unofficial format.

**Comment by fenhl** [comment]
Canadian Highlander support is implemented on the Lore Seeker branch now, with automated points list updates. If you're interested I can send a PR.

**Comment by taw** [comment]
It looks like what I did to support `FormatAny` (for `restricted:* OR banned:*` and such) might conflict with your Canadian Highlander code, but it should be easy fix.

I see that you have a lot of changes, not just related to custom cards. Would it be worth going through them and seeing which ones would be worth merging back?
